client/src/components/TripElement/StageForm.js

```diff
@@ -0,0 +1,51 @@
+import { TextField } from '@mui/material'
+import PropTypes from 'prop-types'
+import { useForm } from 'react-hook-form'
+import { useSelector } from 'react-redux'
+import { Button } from '../common'
+
+StageForm.propTypes = {
+  onSubmit: PropTypes.func.isRequired,
+}
+
+export function StageForm({ onSubmit }) {
+  const stagesByDay = useSelector((state) => state.stages.stages)
+  const editStageId = useSelector((state) => state.stages.editStageId)
+  const stage = stagesByDay.flat().find((item) => item._id === editStageId)
+
+  const { register, handleSubmit } = useForm({
+    defaultValues: {
+      updateNotes: '',
+    },
+  })
+
+  const onSubmitForm = (data) => {
+    onSubmit({ ...data, stage })
+  }
+
+  return (
+    <form
+      onSubmit={handleSubmit(onSubmitForm)}
+      className='flex flex-col space-y-4 text-center'
+    >
+      <div>
+        <div className={'text-2xl'}>
+          {stage.emoji} {stage.stageLocation}
+        </div>
+        <div>{stage.description}</div>
+      </div>
+      <TextField
+        {...register('updateNotes', { required: false })}
+        label='Update'
+        placeholder='Tell me what you would like to change about this...'
+      />
```

Comment:
it works, just wondering if we should allow update with no prompt

Reply:
good point, ill make it required